packages/flutter/lib/fix_data.yaml

```diff
@@ -17,6 +17,65 @@
 
 version: 1
 transforms:
+  - title: 'Remove BoxShape.rectangle'
+    date: 2022-07-27
+    element:
+      uris: [ 'widgets.dart', 'material.dart', 'cupertino.dart' ]
```

Comment:
Isn't BoxDecoration defined in the painting library? I am trying to find where it gets exported from there, but that may have to do with the fix not working here.

Reply:
The uris here should include every library that exports BoxDecoration. then, each `.dart/.dart.expect` file associated with those uris should contain the sample code to validate each.